src/electron/views/device-connect-view/components/window-title.tsx

```diff
@@ -1,32 +1,48 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+import { WindowStateStoreData } from 'electron/flux/types/window-state-store-data';
+import { PlatformInfo } from 'electron/platform-info';
 import { isEmpty } from 'lodash';
 import * as React from 'react';
 import { NamedFC } from '../../../../common/react/named-fc';
-import { actionableIconsContainer, headerText, titleContainer, windowTitle } from './window-title.scss';
+import { actionableIconsContainer, headerText, macWindowTitle, titleContainer, windowTitle } from './window-title.scss';
 
+export interface WindowTitleDeps {
+    platformInfo: PlatformInfo;
+}
 export interface WindowTitleProps {
+    deps: WindowTitleDeps;
     title: string;
     children?: JSX.Element;
     actionableIcons?: JSX.Element[];
     className?: string;
+    windowStateStoreData: WindowStateStoreData;
 }
 
 export const WindowTitle = NamedFC<WindowTitleProps>('WindowTitle', (props: WindowTitleProps) => {
+    if (props.windowStateStoreData.currentWindowState === 'fullScreen') {
+        return null;
+    }
+
+    const windowtitleClassNames = [windowTitle, props.className].filter(c => c != null);
+
+    if (props.deps.platformInfo.isMac()) {
+        windowtitleClassNames.push(macWindowTitle);
+    }
     return (
-        <header className={[windowTitle, props.className].filter(c => c != null).join(' ')}>
+        <header className={windowtitleClassNames.join(' ')}>
```

Comment:
so... this line: 
```typescript
 const windowtitleClassNames = [windowTitle, props.className].filter(c => c != null);
```
and this line 
```typescript
windowtitleClassNames.join(' ')
```
is it worth it to use office fabric `css` function? (which does the same)